modules/autostash.lua

```diff
@@ -58,6 +58,20 @@ local function prepare_floaty_text(list, surface, position, name, count)
 end
 
 local function chest_is_valid(chest)
+   if this.dungeons_initial_level ~= nil then
+      -- game.print('ERIC cpos ' .. serpent.line(chest.position))
```

Comment:
This does not belong here. Don't leave any comments.

Reply:
I think that this autostash code is not needed in general, allowing the transport chests as valid containers is probably not the best idea in general (players should have good control what goes into them, and there are loaders available in most of scenarios), but even then, including blue chest entity type in allowed chests would be clever way (even if conditional for one map) instead of checking for coordinates of each scanned chest.

Reply:
This approach seemed to match with the way people tended to play "dump loot into the transport chests so it goes up to a sorter"

I can certainly remove it. In previous runs of the map I found myself manually dumping things into the transport chests since there was no easy way to auto load them. I guess the next closest would be have a chest and a loader and always pick up the loader, empty stuff and then put the loader back.

The reason for specific coordinates rather than all blue chests is to avoid the problem of dumping into blue chests used for logistics factories.

Reply:
Blue Chest is special entity, it only shares graphics with the logistic blue chest. It is creative-only chest (same as red chest, both are used for transporting stuff for that unique reason). So if we do "if dungeons then blue chest is whitelisted", then it is one operation and it will automatically find the chest.

The whitelist still is just first step, the "if chest is used (e.q. has inserters), that still can discard the chest again from being eligible.

Reply:
oh, I didn't realize those chests were a different entity. Yea, should be easy to switch over to filter for those.  I still want to ignore having inserters/loaders on them since you will but dump to those is really nice.

Reply:
Fixed to use red-chest and blue-chest in 7cc268fab6a2342d52d60b6366cdf0138d93bca9